src/Psalm/Internal/Analyzer/Statements/Expression/Call/Method/MethodCallReturnTypeFetcher.php

```diff
@@ -449,7 +450,7 @@ public static function taintMethodCallResult(
                 $stmt_var_type = $context->vars_in_scope[$var_id]->setParentNodes(
                     $var_nodes,
                 );
-                
+
```

Comment:
sorry, I am unable to revert this change, my IDE is auto removing these whitespaces and restoring them is more annoying than just having these deleted. I hope thats okay.

Reply:
That's totally fine.